src/Symfony/Component/Validator/ValidatorBuilder.php

```diff
@@ -152,7 +152,7 @@ public function addYamlMappings(array $paths): static
     /**
      * Adds a list of classes with mapping constraints as attributes.
      *
-     * @param class-string[] $classes The paths to the mapping classes
+     * @param array<class-string, class-string[]> $classes
```

Comment:
should be defined as `array<class-string, list<class-string>>`, to ensure that the recursive merge produces the expected type, as the recursion behavior is totally different for lists and non-list arrays